app/src/main/java/com/vci/vectorcamapp/intake/presentation/IntakeScreen.kt

```diff
@@ -576,22 +551,13 @@ fun IntakeScreen(
                     surveillanceForm.numPeopleSleptUnderLlin?.let { current ->
```

Comment:
current is unused. Can we replace it with numPeopleSleptUnderLlin, and then replace accordingly in line 554?